src/main/webapp/scripts/account-info.js

```diff
@@ -0,0 +1,14 @@
+var loggedInStatus;
+var logInOutURL;
+var signupURL;
+
+function updateAccountData(){
```

Comment:
For all methods in the script file I would suggest including a description of the method.

Reply:
updated